docs/introduction-to-smart-contracts.rst

```diff
@@ -51,8 +51,9 @@ code that manages the database. In this example, the contract defines the
 functions ``set`` and ``get`` that can be used to modify
 or retrieve the value of the variable.
 
-To access a state variable, you can not prefix with ``this.`` as is common in
-other languages.
+To access a state variable, you do not typically add the ``this.`` prefix.
+Unlike in some other languages, omitting it is not just a matter of style,
+it results in a completely different operation.
```

Comment:
```suggestion
To access a member (like a state variable) of the current contract, you do not typically add the ``this.`` prefix,
you just access it directly via its name.
Unlike in some other languages, omitting it is not just a matter of style,
it results in a completely different way to access the member, but more on this later.
```

Reply:
Not sure if this is better...

Reply:
Sorry that this discussion is dragging on for so long, but the early chapters in the documentation are often the most difficult ones.

Reply:
I think it's fine too but let's see what is @gqqnbig's opinion.